scripts/generate_docs.sc

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env amm
```

Comment:
I wonder if this should be in `ci`. What do you think?

Reply:
It should eventually be a jenkins job :)